### ISSUE TITLE
api_client Undefined

### ISSUE THREAD

**Issue by jbcayrou** [fabriceb/sfFacebookConnectPlugin#2]
In the file sfFacebookConnect.class.php 
for example line 30 :
$rows = sfFacebook::getFacebookClient()->api_client->fql_query($query);
But with the new sdk facebook, attribute api_client doesn't exist.
It's the same problem line 178,179,184,201

**Comment by fabriceb** [comment]
Indeed, I merged quickly Benjamin's branch and the new Facebook API brought some bugs. I will correct them !

**Comment by jbcayrou** [comment]
Thank you very much !

**Comment by patryk** [comment]
I've fixed this. I've sent you pull request (not sure if correctly). Pull my change.

**Comment by fabriceb** [comment]
I did not receive your pull request :-(

**Comment by tobiassjosten** [comment]
The changes below solved this problem. Thanks, patryk!

https://github.com/fabriceb/sfFacebookConnectPlugin/issues/#issue/6
